backend/src/test/java/ca/gov/dtsstn/cdcp/api/web/v1/model/mapper/SubscriptionModelMapperTest.java

```diff
@@ -21,14 +19,27 @@ class SubscriptionModelMapperTest {
 	SubscriptionModelMapper subscriptionModelMapper = Mappers.getMapper(SubscriptionModelMapper.class);
 
 	@Test
+	@DisplayName("Test subscriptionModelMapper.toModel() w/ null subscriptions")
 	final void testToModelStringIterableOfSubscriptionNullSubscription() {
 		final Collection<Subscription> subscriptions = null;
 		assertThat(subscriptionModelMapper.toModel("d827416b-f808-4035-9ccc-7572f3297015", subscriptions)).isEmpty();
 	}
+
 	@Test
+	@DisplayName("Test subscriptionModelMapper.toModel() w/ non-null subscriptions")
 	final void testToModelStringIterableOfSubscriptionNonNullSubscription() {
-		final Collection<Subscription> subscriptions = Stream.of(ImmutableSubscription.builder().id("00000000-0000-0000-0000-000000000000").build(), ImmutableSubscription.builder().id("22222222-2222-2222-2222-222222222222").build()).collect(Collectors.toCollection(HashSet::new));
-		assertThat(subscriptionModelMapper.toModel("d827416b-f808-4035-9ccc-7572f3297015", subscriptions)).isNotEmpty();
+		final Iterable<Subscription> subscriptions = List.of(
+				ImmutableSubscription.builder()
+					.id("00000000-0000-0000-0000-000000000000")
+					.build(),
+				ImmutableSubscription.builder()
+					.id("22222222-2222-2222-2222-222222222222")
+					.build());
+		final var subscriptionModels = subscriptionModelMapper.toModel("d827416b-f808-4035-9ccc-7572f3297015", subscriptions);
+		assertThat(subscriptionModels)
+			.hasSize(2)
+			.anyMatch(model -> model.getId() == "00000000-0000-0000-0000-000000000000")
+			.anyMatch(model -> model.getId() == "22222222-2222-2222-2222-222222222222");
```

Comment:
Strings have to be compared with `.equals()`

(🤣 doesn't sonarlint warn you about this? 🤣)